### PR TITLE
add support for  WinServer2008R2sp1 (issue #84)

### PR DESCRIPTION
**Pull request by FroggieFrog**
Hopefully everything works correct...

### CODE REVIEW

File: Documentation/Content/OsFilters.aml [hunk 1]
```diff
@@ -179,6 +179,10 @@
               <entry>Windows Server 2008 R2</entry>
               <entry>winServer2008R2</entry>
             </row>
+			<row>
```
Comment (dblock) [inline Documentation/Content/OsFilters.aml]: Sup' with the formatting? ;) Mind fixing this one please?

File: Documentation/Content/OsFilters.aml [hunk 2]
```diff
@@ -202,7 +206,7 @@
             <row>
               <entry>Windows Server 2012 R2</entry>
               <entry>win8serverR2</entry>
-            </row>
+            </row>
```
Comment (dblock) [inline Documentation/Content/OsFilters.aml]: Remove the trailing spaces added ...

**Comment by dblock** [comment]
This needs an entry in CHANGELOG and please fix tabs vs. spaces to make the diff look clean. Thanks.

**Comment by FroggieFrog** [comment]
I hope now is everything correct.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,7 @@ Features
 
 * [#46](https://github.com/dblock/dotnetinstaller/pull/46) - Added support for hiding the application window for ShellExecute execution method for command components - [@icnocop](https://github.com/icnocop).
 * [#75](https://github.com/dblock/dotnetinstaller/issues/75) - Added the ability to set the current working directory for command components - [@icnocop](https://github.com/icnocop).
+* [#84](https://github.com/dblock/dotnetinstaller/issues/84) - Is it possible to add WinServer2008R2sp1 to the list of Operating Systems? - [@FroggieFrog](https://github.com/FroggieFrog).
```
Comment (dblock) [inline CHANGELOG.md]: This is a changelog, so it should say "Added support for Windows Server 2008R2 SP1" or something like that.

**Comment by dblock** [comment]
Looks like you also need to modify https://github.com/dblock/dotnetinstaller/blob/master/Documentation/Content/OsFilters.aml, please. Double-check whether it has anything missing too. Appreciated.

**Comment by dblock** [comment]
Please also squash your commits. Thx.

**Comment by icnocop** [comment]
I merged this pull request into this one: https://github.com/dblock/dotnetinstaller/pull/87

**Comment by dblock** [comment]
Thanks @icnocop

**Pull request closed without merging**